<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WarehouseOut</class>
 <widget class="QMainWindow" name="WarehouseOut">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>410</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>410</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>580</width>
    <height>410</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="SpareView">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>140</y>
      <width>161</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QListView{  /*常态*/
	background-color: rgb(204, 221, 255);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QLabel" name="Label1">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>总库存量:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>出库时间:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="OutTime">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>260</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{  /*常态*/
	background-color: rgb(204, 221, 255);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>yyyy-MM-dd hh:mm:ss</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CabinetEdit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{  /*常态*/
	background-color: rgb(204, 221, 255);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Label5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>存放柜号:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OutButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>320</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{  /*常态*/
	color:white;
	background-color: rgb(85, 170, 255);
	border-radius: 20px;  border: 2px groove lavender;
	border-style: outset;
}
QPushButton:hover{  /*鼠标悬浮时*/
	background: rgb(49,106,234);
}
QPushButton:pressed{  /*鼠标按下时*/
	background:rgb(96,96,96);
	color: rgb(200, 200, 200);
}</string>
    </property>
    <property name="text">
     <string>出库</string>
    </property>
   </widget>
   <widget class="QLabel" name="TotalNum">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ReturnButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>320</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{  /*常态*/
	color:white;
	background-color: rgb(85, 170, 255);
	border-radius: 20px;  border: 2px groove lavender;
	border-style: outset;
}
QPushButton:hover{  /*鼠标悬浮时*/
	background: rgb(49,106,234);
}
QPushButton:pressed{  /*鼠标按下时*/
	background:rgb(96,96,96);
	color: rgb(200, 200, 200);
}</string>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QLabel" name="BackGround">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>571</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255,255,255);
border-radius: 15px;
border-style: outset;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LogoLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>200</width>
      <height>53</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../resources.qrc">:/images/logo200x53.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="SaveNum">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>90</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Label2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>90</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>安全库存:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SpareEdit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{  /*常态*/
	background-color: rgb(204, 221, 255);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Label4">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>140</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>备件号:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label7">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>320</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>数量:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="OutNum">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>320</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 221, 255);
border: none;
border-radius: 2px;
color: rgb(0,0,0);
font: 16pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="Label3">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>110</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>选择冲压备件⬋</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ExportButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>150</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{  /*常态*/
	color:white;
	background-color: rgb(85, 170, 255);
	border-radius: 20px;  border: 2px groove lavender;
	border-style: outset;
}
QPushButton:hover{  /*鼠标悬浮时*/
	background: rgb(49,106,234);
}
QPushButton:pressed{  /*鼠标按下时*/
	background:rgb(96,96,96);
	color: rgb(200, 200, 200);
}</string>
    </property>
    <property name="text">
     <string>记录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangeButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{  /*常态*/
	color:white;
	background-color: rgb(85, 170, 255);
	border-radius: 10px;  border: 2px groove lavender;
	border-style: outset;
}
QPushButton:hover{  /*鼠标悬浮时*/
	background: rgb(49,106,234);
}
QPushButton:pressed{  /*鼠标按下时*/
	background:rgb(96,96,96);
	color: rgb(200, 200, 200);
}
QPushButton:!enabled {   /*按钮禁用时*/
	background: rgb(100, 100, 100);
	color: rgb(200, 200, 200);
}
</string>
    </property>
    <property name="text">
     <string>切换为成型备件</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="MoldCombo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{  /*常态*/
	background-color: rgb(204, 221, 255);
	border: none;
	border-radius: 2px;
	color: rgb(0,0,0);
	font: 18pt &quot;微软雅黑&quot;;
}</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>BackGround</zorder>
   <zorder>SpareView</zorder>
   <zorder>Label1</zorder>
   <zorder>Label6</zorder>
   <zorder>OutTime</zorder>
   <zorder>CabinetEdit</zorder>
   <zorder>Label5</zorder>
   <zorder>OutButton</zorder>
   <zorder>TotalNum</zorder>
   <zorder>ReturnButton</zorder>
   <zorder>LogoLabel</zorder>
   <zorder>SaveNum</zorder>
   <zorder>Label2</zorder>
   <zorder>SpareEdit</zorder>
   <zorder>Label4</zorder>
   <zorder>Label7</zorder>
   <zorder>OutNum</zorder>
   <zorder>Label3</zorder>
   <zorder>ExportButton</zorder>
   <zorder>ChangeButton</zorder>
   <zorder>MoldCombo</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>580</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>ChangeButton</tabstop>
  <tabstop>MoldCombo</tabstop>
  <tabstop>SpareView</tabstop>
  <tabstop>SpareEdit</tabstop>
  <tabstop>CabinetEdit</tabstop>
  <tabstop>OutTime</tabstop>
  <tabstop>OutNum</tabstop>
  <tabstop>OutButton</tabstop>
  <tabstop>ReturnButton</tabstop>
  <tabstop>ExportButton</tabstop>
 </tabstops>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
